DSharpPlus/DiscordClient.cs

```diff
@@ -1837,7 +1837,7 @@ internal Task OnResumedAsync()
                 }
                 else
                 {
-                    mentioned_users = Utilities.GetUserMentions(message).Select(xid => this.InternalGetCachedUser(xid)).ToList();
+                    mentioned_users = Utilities.GetUserMentions(message).Select(this.InternalGetCachedUser).ToList();
```

Comment:
Never did/see that. Does that actually work?

Reply:
yes

Reply:
welcome to java world, we have method references

Reply:
It's valid is a predicate. It just doesn't grow the callstack as much.